tests/ut/ir/transforms/test_dsa_reuse_penalty_recognizer.py

```
@@ -505,17 +505,17 @@ def main(
             kvalue: pl.Scalar[pl.INT32],
             output: pl.Tensor[[16, 8], pl.INT32],
         ) -> pl.Tensor[[16, 8], pl.INT32]:
-            source = pl.load(source_tensor, [0, 0], [16, 64])
-            temporary = pl.tile.create([16, 64], pl.UINT8)
+            source = pl.load(source_tensor, [0, 0], [16, 64], target_memory=pl.Mem.Vec)
+            temporary = pl.tile.create([16, 64], pl.UINT8, target_memory=pl.Mem.Vec)
             destination, _count = pl.tile.gather_compare(
```

Ruff (0.16.1)

[warning] Unpacked variable `destination` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)